docker/Dockerfile.cpu

```diff
@@ -1,4 +1,6 @@
-FROM tensorflow/serving:1.11.1 as tfs
+ARG TFS_VERSION
+
+FROM tensorflow/serving:${TFS_VERSION} as tfs
```

Comment:
There doesn't exist a base EI binary image.

Right now EI handles installation by copying the binary over from the local host.

The COPY --from doesn't get interpreted properly by Docker for local files. We would probably have to introduce a new arg?